src/components/assets/Assets.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <div class="wrapper--assets">
+    <div v-if="!selectedAddress" class="backdrop--transparent" />
```

Comment:
![Screen Shot 2022-03-21 at 9 32 22 AM](https://user-images.githubusercontent.com/40356749/159227643-f003bbe2-bbb5-49fa-8668-732bc6224960.png)

When I first access the page, the assets are blurred, and there are no further instructions from the page. What about opening the wallet connect modal when the user accesses the page?

Reply:
In that case, the sidebar will be opened automatically. The logic will be implemented once the `select wallet` component (global) has been implemented by @sirius651 .

https://gyazo.com/9dd2e3ea8f58875a7cbb7e81a43287c2